lib/utils.js

```diff
@@ -41,7 +41,8 @@ export function isMultilanguageGrammar(grammar: atom$Grammar) {
 export function getEmbeddedScope(
   editor: atom$TextEditor,
   position: atom$Point
-): ?string {
+): string {
+  // Flow had a problem with returning ?string
```

Comment:
I'm pretty sure `?string` is correct here since `_.find(scopes, s => s.indexOf("source.embedded.") === 0)` can return undefined.

We probably need to account for that case somewhere else.